src/convertToObject.js

```diff
@@ -6,7 +6,29 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+
+  const rules = sourceString.split(';');
```

Comment:
The variable name 'rules' could be more descriptive. Consider renaming it to 'styleDeclarations' or 'cssRules' to better convey its purpose.